tests/fields/test_custom.py

```diff
@@ -0,0 +1,9 @@
+import strawberry
+
+
+strawberry.field
+
+
+def test_override_get_result():
+    # TODO: after patrick approves changes.
```

Comment:
Let's make sure not to merge until this is resolved

Reply:
I'll add an assertion